### DISCUSSION TITLE
Keep original device partition name case on Samba NAS

### DISCUSSION THREAD

**Discussion by ghost54770** [dianlight/hassio-addons#279] [Ideas]
Sorry for my english.
I think is a good idea to create a setting to keep the current partition name case (usb device). I use frigate addon for Home Assisant on a raspberry and his default media folder is /media/frigate but Samba NAS convert partition name with only uppercase (/media/FRIGATE). I must use Samba NAS  to mount external USB device (because it's not terrible to use mount command on usb device on Home Assisant).

**Comment by dianlight** [comment]
There was a lot of requests on this topic but I think it's important to understand that the incompatibility between these addon and the frigate addon can be resolved on both sides.
I don't know how frigate addon works and why is fixed on lowercase path but on sambanas site the use of uppercase name is derived from how Windows implementation of cifs protocol works and is derived from fat ( yes old 8bit fat! ). 

My never answered question is 'why the frigate addons use lowercase and don't allow to change?'

However in the next major version I can add a configuration to manage that if there are sufficient requests. 

So for all the people interested in this feature set a like o vote this issue.

L.

**Comment by BobMac57** [comment]
I just up voted this... I am using a symlink for now but I'd prefer not to.

**Reply by martinwierenga** [reply]
Did you get symlink to be persistent? I rather mount the drives with SambaNAS (purely use it's mounting feature and symlink the necessary folders).
